html/changelogs/lohikar-lights.yml

```diff
@@ -0,0 +1,4 @@
+author: Lohikar
+delete-after: True
+changes:
+  - bugfix: "Engineering crews have corrected a fault in station wiring that caused station lighting to be instead connected to the Equipment power pool. Officials suspect a prank from a misbehaving maintenance drones."

```

Comment:
Because I can't let any PR go without a review, technically it should be `a maintenance drone`. ;)

Reply:
No, it should be `from maintenance drones.`. I only half-converted it to plural.